### ISSUE TITLE
KeyError: u'USLACKBOT'

### ISSUE THREAD

**Issue by magdics** [hfaran/slack-export-viewer#125]
Can't seem to get slack-export-viewer to read our exports:  please advise.

File "/Library/Frameworks/Python.framework/Versions/2.7/bin/slack-export-viewer", line 11, in <module>
    sys.exit(main())
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/slackviewer/main.py", line 50, in main
    configure_app(app, archive, debug)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/slackviewer/main.py", line 25, in configure_app
    top.dm_users = reader.compile_dm_users()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/slackviewer/reader.py", line 68, in compile_dm_users
    dm_members = {"id": dm["id"], "users": [self.__USER_DATA[m] for m in dm["members"]]}
KeyError: u'USLACKBOT'

**Comment by semperrabbit** [comment]
I had an issue with this previously, but hacked my way around it late 2017. That hack subsequently broke after syncing with upstream in early '18. 
Tldr: pretty sure it's bc slack formally decommissioned it, so the username doesn't exist and will not show up in the user list. Also, the messages associated arw a different format with additional fields including the IRC username that originated the msg.

**Comment by hfaran** [comment]
Resolved by #133
